### DISCUSSION TITLE
Next contributor meetup planning

### DISCUSSION THREAD

**Discussion by NiteshKant** [netty/netty#11744] [General]
Let us use this discussion for discussing the time and agenda for the next contributor meetup.

**_Tentative date_**: Oct 28, 2021
**_Time_**: 8:30 am pacific time

**_Agenda_**

1. Go over the new `Buffer` API. 
(_This was requested in the last meetup because the API has changed since it was discussed the first time._)
2. [TBD ... Please propose if you want to have something discussed.]

**Comment by NiteshKant** [comment]
FYI @netty/contributors

**Comment by hyperxpro** [comment]
2: HTTP Caching (both client and server-side).

**Comment by chrisvest** [comment]
3. Netty Core/Contrib separation. Mentioned in #11555
